### PR TITLE
sort button

### PR DESCRIPTION
**Pull request by hellokathylee**
- modify sort button on main view (switches between time and priority)
- modify welcome message on main view

### CODE REVIEW

File: Uni_life_tracker/app/src/main/res/layout/activity_main_page.xml [hunk 1]
```diff
@@ -38,48 +51,41 @@
...
-                android:text="@string/welcome"
-                android:textSize="15sp"
-                android:fontFamily="@font/roboto_mono_light"
                 android:layout_marginTop="5dp"
-                android:layout_marginStart="20dp"/>
+                android:layout_marginHorizontal="20dp"
+                android:gravity="center_vertical">
 
-        <!--  Welcome Name  -->
-        <TextView
```
Comment (EvgeniaSila) [inline Uni_life_tracker/app/src/main/res/layout/activity_main_page.xml:64]: Would it be possible to move up the welcome message just so the spacing isn't too tight between the date and the welcome text?

**Review by EvgeniaSila — CHANGES_REQUESTED** [review]

**Review by EvgeniaSila — DISMISSED** [review]
Looks good

File: Uni_life_tracker/app/src/main/res/layout/activity_main_page.xml (hunk 1 above)
Comment (hellokathylee) [reply Uni_life_tracker/app/src/main/res/layout/activity_main_page.xml:64]: yup, added spacing to it!

**Review by EvgeniaSila — APPROVED** [review]

File: Uni_life_tracker/app/src/main/java/com/generic/plannr/MainPageActivity.java [hunk 2]
```diff
@@ -6,7 +6,7 @@
 import android.app.AlertDialog;
 import android.content.Intent;
 import android.view.View;
-import android.widget.TextView;
+import android.widget.*;
```
Comment (dalshekerchi) [inline Uni_life_tracker/app/src/main/java/com/generic/plannr/MainPageActivity.java:9]: do we need to import all widgets, can we just import the ones we need?

**Review by dalshekerchi — CHANGES_REQUESTED** [review]

**Review by dalshekerchi — APPROVED** [review]

File: Uni_life_tracker/app/src/main/java/com/generic/plannr/MainPageActivity.java (hunk 2 above)
Comment (hellokathylee) [reply Uni_life_tracker/app/src/main/java/com/generic/plannr/MainPageActivity.java:9]: we would have to make 5 imports

**Pull request merged**